common/compositor/compositor.cpp

```diff
@@ -40,19 +40,22 @@ Compositor::~Compositor() {
 }
 
 void Compositor::Init() {
-  gpu_resource_handler_.reset(CreateNativeGpuResourceHandler());
```

Comment:
renderer_ need to be initialized in Compositor::Init(). If not, below call will caused fault address error.
renderer_->SetExplicitSync(ExplicitSyncEnabled);